controllers/extendeddaemonsetreplicaset/strategy/limits/limits.go

```diff
@@ -8,13 +8,13 @@ package limits
 
 // Parameters use to provide the parameters to the Calculation function.
 type Parameters struct {
-	NbNodes             int
-	NbPods              int
-	NbAvailablesPod     int
-	NbOldAvailablesPod  int
-	NbCreatedPod        int
-	NbUnresponsiveNodes int
-	NbUnreadyPods       int
+	NbNodes              int
+	NbPods               int
+	NbAvailablesPod      int
+	NbOldAvailablesPod   int
+	NbCreatedPod         int
+	NbUnresponsiveNodes  int
+	NbUnavailableOldPods int
```

Comment:
for consistency with the parameter above could we call it `NbOldUnavailablePods` ? (i just realized that we should probably change `Nb*AvailablesPod` ->  `Nb*AvailablePods` but can be done in a different PR)